package.json

```diff
@@ -50,7 +50,9 @@
     "@popperjs/core": "2.11.5",
     "@react-hook/merged-ref": "1.3.2",
     "@react-stately/toggle": "3.2.7",
+    "a11y-dialog": "^7.5.0",
     "autosize": "^5.0.1",
+    "body-scroll-lock": "^4.0.0-beta.0",
```

Comment:
are we ok with using the beta version?

Reply:
+1

Reply:
yes, that's the current version, that's what you get when you do `npm i body-scroll-lock`.